fable/include/fable/json.hpp

```diff
@@ -26,6 +26,14 @@
 #ifndef FABLE_JSON_HPP_
 #define FABLE_JSON_HPP_
 
+#ifndef NLOHMANN_JSON_WITH_COMMENTS
```

Comment:
I was wondering what `WITH_COMMENTS` actually means. To me, something like `ALLOW_COMMENTS` would be more descriptive.

Reply:
Good suggestion!

Reply:
Done.